docs/2.7.0/bin/live-preview

```diff
@@ -24,5 +24,16 @@ $DIR/setup-sphinx-source-tree
 DATE=$(date +"%Y-%m-%d")
 echo { \"$DATE\" : \"$DATE\" } >  $BUILD_DIR/gen/versions.json
 
+## Canton Snippets workaround
```

Comment:
Possible improvement:
```
## Canton Snippets workaround: Enable Sphinx extension ignoring the `snippet` directive
```

If possible, extracting this code block in some sort of "function" which has a descriptive name would be best for posterity.

For example, `function disable_canton_snippet_directive`